src/test/java/org/skife/jdbi/v2/sqlobject/TestObjectMethodBehavior.java

```diff
@@ -21,13 +21,15 @@
 import java.sql.Connection;
 import java.sql.SQLException;
 
+import static org.junit.Assert.assertEquals;
+
 /**
  * Sometimes the GC will call {@link #finalize()} on a SqlObject from
- * extremely sensitive places from within the GC machinery.  JDBI should not
- * open a {@link Connection} just to satisfy a (no-op) finalizer.
+ * extremely sensitive places from within the GC machinery.
  * <a href="https://github.com/brianm/jdbi/issues/82">Issue #82</a>.
+ * Other methods also do not use a {@link Connection} and can cause issues acquiring one.
  */
-public class TestFinalizeBehavior
+public class TestObjectMethodBehavior
```

Comment:
Maybe it's a better idea to write a new test instead changing the finalize test?